test/blackbox-tests/test-cases/pkg/ocamlformat-dev-tool.t

```diff
@@ -0,0 +1,487 @@
+Exercises end to end, locking and building ocamlformat dev tool.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+Make a fake ocamlformat:
```

Comment:
should this be OCamlFormat or `ocamlformat`?

Reply:
OCamlFormat.